packages/sit-onyx/src/components/OnyxListboxOption/OnyxListboxOption.ct.tsx

```diff
@@ -8,12 +8,17 @@ test.describe("Screenshot tests", () => {
       name: `Listbox option (${state})`,
       columns: ["default", "selected"],
       rows: ["default", "hover", "focus-visible"],
+      // TODO: color-contrast: remove when contrast issues are fixed in https://github.com/SchwarzIT/onyx/issues/410
+      // aria-required-parent is ignored here because this component is only a single option which is internally always
+      // used together with a parent so we disable the failing rule here
+      disabledAccessibilityRules: ["aria-required-parent", "color-contrast"],
       component: (column, row) => (
         <OnyxListboxOption
           aria-label="Label"
           aria-selected={column === "selected"}
           active={row === "focus-visible"}
           aria-disabled={state === "disabled"}
+          {...{ role: "option" }}
```

Comment:
does this syntax not work? :O
```suggestion
          role="option"
```

Reply:
Unfortunately no, it throws a type error :( Thats why I did it in this "ungly" way

Reply:
oki